pipeline/container/manager.py

```diff
@@ -219,8 +219,8 @@ def run(
     ) -> t.Any:
         with logger.contextualize(run_id=run_id):
             logger.info("Running pipeline")
-            args = self._parse_inputs(input_data, self.pipeline)
             try:
+                args = self._parse_inputs(input_data, self.pipeline)
```

Comment:
I think this line could have stayed where it was before?

Otherwise all the `RunInputException`s will be caught and re-raised as `RunnableErrors`.
I guess you can test this though

Reply:
I think this was causing the original issue which is that an input parsing error results in a `500`, and no logs for the run that the user can see. Discussed with Yvan maybe returning a proper HTTP error, we've made a ticket for it here: [PC-952](https://linear.app/mystic-ai/issue/PC-952/pipeline-container-run-error-refinements)

Reply:
Nvm, as pointed out, we check the exception type and will specifically return a 400 for a run input exception.